src/helpers/CommonCLI.cpp

```diff
@@ -129,12 +299,28 @@ void CommonCLI::savePrefs() {
 }
 
 void CommonCLI::handleCommand(uint32_t sender_timestamp, const char* command, char* reply) {
-    if (memcmp(command, "reboot", 6) == 0) {
+    // Split help to keep messages short
+    // device, radio, mesh, auth, debug
+    if (memcmp(command, CMD_HELP, 4) == 0) {
+      if (memcmp(command, "help device cmds", 16) == 0) { format_help_device_cmds(reply); return; }
+      if (memcmp(command, "help device keys", 16) == 0) { format_help_device_keys(reply); return; }
+      if (memcmp(command, "help radio cmds", 15)  == 0) { format_help_radio_cmds(reply);  return; }
```

Comment:
It would probably make sense to use the constants here as well, something like:

```suggestion
      if (memcmp(command, CMD_HELP + SPACE + KEY_RADIO + "cmds", 15)  == 0) { format_help_radio_cmds(reply);  return; }
```